explorer/app/components/Index/common/utils.ts

```diff
@@ -1,5 +1,127 @@
 // App dependencies
 import { MetadataValue } from "./entities";
+import { SummaryResponse } from "../../../models/responses";
+
+/**
+ * Calculates the summary file format count using count values returned for each file format in the summary response.
+ * @param summaryResponse - Response model return from summary API.
+ * @returns count of file formats.
+ */
+export function calculateSummaryFileFormatsCount(
+  summaryResponse: SummaryResponse
+): number {
+  return (summaryResponse.fileFormats ?? []).reduce((accum, { count }) => {
+    return accum + count;
+  }, 0);
+}
+
+/**
+ * Calculates the summary total cell count using the estimatedCellCount and totalCells values fom the summary response.
+ * @param summaryResponse - Response model return from summary API.
+ * @returns count of total cell count.
+ */
+export function calculateSummaryTotalCellCount(
+  summaryResponse: SummaryResponse
+): number {
+  return (summaryResponse.projects ?? []).reduce(
+    (accum, { cellSuspensions, projects }) => {
+      if (
+        projects &&
+        (projects.estimatedCellCount || projects.estimatedCellCount === 0)
+      ) {
+        accum += projects.estimatedCellCount;
+      } else if (
+        cellSuspensions &&
+        (cellSuspensions.totalCells || cellSuspensions.totalCells === 0)
+      ) {
+        accum += cellSuspensions.totalCells;
+      }
+      return accum;
+    },
+    0
+  );
+}
+
+/**
+ * Formats count sizes.
+ * @param value - Count size.
+ * @returns formatted count size as display string.
+ */
+export function formatCountSize(value: number): string {
```

Comment:
F, just checking, is this the same functionality as the Angular version?

Reply:
It is the pipe code, copied and unmodified. I can double check it though?

Reply:
I updated to constant `precision` from `let precision = 1` to `const precision = 1`.